src/services/NetworkService/realtimeUpdates.ts

```diff
@@ -1,33 +1,59 @@
 import { constants } from '../AppService';
 import { io, Socket } from 'socket.io-client';
+import { SdkManager } from '../common';
 
 export class RealtimeUpdates {
-  private socket: Socket;
+  private socket!: Socket;
 
-  constructor() {
-    const URL = `${constants.DRIVE_API_URL}`;
-    this.socket = io(URL, {
+  constructor(private sdk: SdkManager) {}
+
+  init() {
+    this.socket = io(constants.NOTIFICATIONS_URL, {
+      auth: {
+        token: this.sdk.getApiSecurity().photosToken,
+      },
       transports: ['websocket'],
```

Comment:
remove this, as the we use the default transport

Reply:
doesnt work well on mobile, seems like it defaults to xhr polling, and it throws connection errors